### PR TITLE
Bootstrap 3

### PR DESCRIPTION
**Pull request by erichoog**
@andy5995 thanks for the bug fix, i was able to test properly now.

This is an implementation of bootstrap 3 (v4 is beta - i forgot).

Currently I'm still using the CDN's as it's conveient and sovles the issue of having to copy the css file to the tmp directory.  So I have delete the CSS file in this branch but we can always add it back. I just wanted to give you an example of what it could look like with bootstrap 3 implemented.

bootstrap3

Let me know what you think and we can go from there.  I should probably close the other pull request as well.

### CODE REVIEW

**Comment by erichoog** [comment]
There are also a few <br /> tags that i will remove

File: src/mhwkb_md2html.c [hunk 1]
```diff
@@ -251,7 +251,7 @@ main (int argc, char **argv)
           if (i < tag_ctr - 1)
             strcat(article_links, ", ");
           else
-            strcat(article_links, "<br /><br />\n");
```
Comment (andy5995) [inline src/mhwkb_md2html.c]: yeah, these wouldn't be needed with CSS would they

**Comment by andy5995** [comment]
Thanks for presenting this @erichoog . It looks really good. But, I'll go back to a quote from the other PR...

>But I'd like to hear more feedback from others first. Can you open a ticket...

**Comment by erichoog** [comment]
You could also do something like this with the boostrap (can do it without bootstrap as well but it's just easier with it).

Here's I made the first 3 articles show up in a row with 3 columns:

image

There would likely need to be a change to the C code for that to wrap 3 articles in `<div class="row"></div>`

The `<div class="col-md-6></div>` could just be added to the article template.

@andy5995 No worries on the part below, just wanted to see if that was something you guys would be looking for.  No worries if not!

> But I'd like to hear more feedback from others first. Can you open a ticket...

**Comment by andy5995** [comment]
>@andy5995 No worries on the part below, just wanted to see if that was something you guys would be looking for. No worries if not!

Right, that's why I suggested a ticket, rather than a discussion on the PR. Tickets are more visible and it keeps things more organized.

**Comment by andy5995** [comment]
@erichoog 
>So I have delete the CSS file in this branch but we can always add it back.

Better to put the css file back, and instead of deleting the line from head, just comment it out, like this:
```
<!-- <link rel="stylesheet" type="text/css" href="mhwkb_style.css"> -->
```

**Comment by erichoog** [comment]
@andy5995 done!

**Comment by andy5995** [comment]
But also please un-delete our mhwkb_style.css file :)

**Comment by erichoog** [comment]
Done!

File: templates/index.html [hunk 2]
```diff
@@ -4,13 +4,27 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>{{data.title}}</title>
-    <link rel="stylesheet" type="text/css" href="mhwkb_style.css">
+    <!-- <link rel="stylesheet" type="text/css" href="mhwkb_style.css"> -->
```
Comment (andy5995) [inline templates/index.html]: Ok, I think it'll be safe to completely delete this line. But don't delete the .css file. Thanks.

File: templates/article_link.html [hunk 3]
```diff
@@ -1 +1 @@
-<a class="tag" href="{{data.link}}">{{data.title}}</a>
+<a class="tag label label-danger" href="{{data.link}}">{{data.title}}</a>
```
Comment (andy5995) [inline templates/article_link.html]: I think this is the problem. No newline at end of file.

And the html being generated, there is an `>` bracket missing after the a
```
">directories</a<br /><br />
```

**Pull request merged**